vignettes/putative_driver_filtering.Rmd

```diff
@@ -0,0 +1,206 @@
+---
+title: >
+  Putative Driver filtering
+author:
+- name: Krutika Gaonkar
+  affiliation: 
+  - &id1 Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia, USA
+  email: gaonkark@chop.email.edu
+- name: Federico Marini
+  affiliation: 
+  - &id2 Institute of Medical Biostatistics, Epidemiology and Informatics (IMBEI), Mainz
+  - Center for Thrombosis and Hemostasis (CTH), Mainz
+  email: marinif@uni-mainz.de
+date: "`r Sys.Date()`"
+package: "`r paste('annoFuse', packageVersion('annoFuse'))"
+output: 
+  html_document:
+    toc: true
+    toc_float: true
+    number_sections: true
+vignette: >
+  %\VignetteIndexEntry{Putative driver filtering}
+  %\VignetteEncoding{UTF-8}  
+  %\VignettePackage{annoFuse}
+  %\VignetteKeywords{Software, Transcription, Visualization, GeneExpression, Annotation,
+    GeneFusionDetection, GUI, ReportWriting}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include=FALSE} 
+knitr::opts_chunk$set(warning = FALSE, message = FALSE)
+```
+
+# Introduction
+
+In this vignette we demonstrate using annoFuse, to filter putative oncogene fusion from a filtered fusion list (filtered fusion list `FilteredFusionAnnoFuse.tsv` was obtained for v16 OpenPBTA files : `fusion_standardization` -> `fusion_filtering_QC` -> `annotate_fusion_calls` ) and also plot recurrent fusion, recurrently fused genes, and summary plot.
```

Comment:
```suggestion
In this vignette, we demonstrate how to use annoFuse to filter putative oncogenic fusions from a filtered fusion list.
The filtered fusion list `FilteredFusionAnnoFuse.tsv` was obtained from the v16 release of [OpenPBTA](https://github.com/AlexsLemonade/OpenPBTA-analysis).
The steps are: `fusion_standardization` -> `fusion_filtering_QC` -> `annotate_fusion_calls`, followed by plotting recurrent fusions, recurrently fused genes, as well as a summary plot.
```

NOTE: `FilteredFusionAnnoFuse.tsv` is not in OpenPBTA - do we need to change this to describe data in the package?

Reply:
I meant to say v16 starfusion , arriba and expression files, the FilteredFusionAnnoFuse.tsv file in created as we run annoFuse functions over these files.

Reply:
ok perfect - you can update then, and also add the release date please